--- /home/book/uboot-original/drivers/net/phy/phy.c	2026-04-12 18:01:44.327796220 -0400
+++ /home/book/my-uboot-v2017.03-4.9.88/drivers/net/phy/phy.c	2026-04-12 09:51:46.004142308 -0400
@@ -221,7 +221,19 @@
 int genphy_update_link(struct phy_device *phydev)
 {
 	unsigned int mii_reg;
-
+#ifdef CONFIG_PHY_SMSC
+    	static int lan8720_flag = 0;
+	int bmcr_reg = 0;
+    if (lan8720_flag == 0) {
+        bmcr_reg = phy_read(phydev, MDIO_DEVAD_NONE, MII_BMCR);
+        phy_write(phydev, MDIO_DEVAD_NONE, MII_BMCR, BMCR_RESET);
+        while(phy_read(phydev, MDIO_DEVAD_NONE, MII_BMCR) & 0X8000) {
+            udelay(100);
+        }
+        phy_write(phydev, MDIO_DEVAD_NONE, MII_BMCR, bmcr_reg);
+        lan8720_flag = 1;
+    }
+#endif
 	/*
 	 * Wait if the link is up, and autonegotiation is in progress
 	 * (ie - we're capable and it's not done)
